packages/teleport/src/services/nodes/makeNode.ts

```diff
@@ -14,25 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { at } from 'lodash';
 import { Node } from './types';
 
 export default function makeNode(json): Node {
-  const [id, clusterId, hostname, tags = [], addr, tunnel = false] = at(json, [
-    'id',
-    'siteId',
-    'hostname',
-    'tags',
-    'addr',
-    'tunnel',
-  ]);
+  const {id, clusterId, hostname, addr, tunnel, tags = [] } = json;
+
+  const tagTexts = tags.map(tag => `${tag.name}: ${tag.value}`)
 
   return {
     id,
     clusterId,
     hostname,
-    tags,
+    tagTexts,
```

Comment:
i would keep the `tag` name as it is shorter. 

```ts
const tags = json.tags || [];
return {
   ...json,
   tags: tags.map(tag => `${tag.name}: ${tag.value}`),
}
```

Reply:
Good idea. Done! :+1: